templates/add-movie.html

```diff
@@ -6,15 +6,15 @@
     <div id="addMovieForm">
 		{{if .FormfillEnabled}}
         <div class="movieInput">
-            <div{{if .ErrTitle}} class="errorMessage"{{end}}><label for="MovieName">Movie Title</label></div>
-			<div><textarea name="MovieName" id="MovieName" style="width:400px">{{if .ValTitle}} {{.ValTitle}}{{end}}</textarea></div>
+            <div{{if .ErrTitle}} class="errorMessage"{{end}}><label class="MovieNameLabel">Movie Name</label></div>
```

Comment:
Just a small thing, but i would prefer that field being called "Movie Title" instead of "Movie Name" (imo Title is the more descriptive word)

Reply:
this was not done by me, i kept all existing names the same, even if i added anything to the name, the original is still in there